spec/std/json/any_spec.cr

```diff
@@ -1,6 +1,8 @@
-require "spec"
+require "../spec_helper"
 require "json"
-require "yaml"
+{% unless flag?(:win32) %}
+  require "yaml"
+{% end %}
```

Comment:
We could do that. But I think this is also fine as it is. It would just add another thing we'd need to look after.
In the prelude this makes sense to keep the list readable, but when it's only an occasional require at a file's start, it's not that useful I think.